src/ec/system76/ec/Kconfig

```diff
@@ -22,3 +22,11 @@ config EC_SYSTEM76_EC_OLED
 	depends on EC_SYSTEM76_EC
 	bool
 	default n
+
+config EC_SYSTEM76_EC_ACPI_DEVICE_HID
+	string
+	default "17761776"
+
+config EC_SYSTEM76_EC_ACPI_DEVICE_NAME
```

Comment:
Kconfig won't allow us to have a bare string without `"`s here, and iASL won't allow us to pass a string type as device name. Let's not add this option.

Reply:
See suggested change below.